### ISSUE TITLE
Full size error

### ISSUE THREAD

**Issue by acastrovargas** [prasathmani/tinyfilemanager#1136]
The full size is not real, it only shows the value of files and not the contents in the folders.

![CinNg8j](https://github.com/prasathmani/tinyfilemanager/assets/30557965/f9e6ea93-e2c2-4b67-bece-65d8b58e95d3)

**Comment by explorerfaketimepass** [comment]
#1100 Check out this PR, as it implements a feature to display disk space usage.

**Comment by acastrovargas** [comment]
The PR is still not integrated?

**Comment by ner00** [comment]
Just note that the script will have to iterate through every sub-folder and its contents to calculate this, every time the page refreshes. I had implemented this on a custom version of mine but had to turn if off due to the performance impact. If you have few files and all with a certain acceptable size, no problem, otherwise the HDD will be taxed immensely.

**Comment by acastrovargas** [comment]
Thanks, I was about to implement, for this reason I will not do it

**Comment by hppanpaliya** [comment]
@ner00 No, the script does not iterate through every sub-folder and its contents to calculate disk space. It uses efficient built-in PHP functions (disk_total_space() and disk_free_space()) that directly query the operating system, so it won't heavily tax the HDD.

**Comment by ner00** [comment]
@hppanpaliya my misunderstanding then, I was thinking of folder file size, which is obviously different and that does have to work in the way I described.

**Comment by acastrovargas** [comment]
![Uploading Captura de pantalla_2024-01-24_15-53-59.png…]()

**Comment by acastrovargas** [comment]
![Uploading Captura de pantalla_2024-01-24_15-53-59.png…]()

**Comment by acastrovargas** [comment]
<a href="https://imgur.com/HNjbCs8.png">
  <img src="https://imgur.com/HNjbCs8.png" />
</a>
